### PR TITLE
elixir v1.9.1

### PR DESCRIPTION
**Pull request by getong**
c0b/docker-elixir#120

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 256d787..5590384 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -86,6 +86,18 @@ elixir:1.8-alpine @ arm64v8
 elixir:1.8-alpine @ i386
 elixir:1.8-alpine @ ppc64le
 elixir:1.8-alpine @ s390x
+elixir:1.8-otp-22 @ amd64
+elixir:1.8-otp-22 @ arm32v7
+elixir:1.8-otp-22 @ arm64v8
+elixir:1.8-otp-22 @ i386
+elixir:1.8-otp-22 @ ppc64le
+elixir:1.8-otp-22 @ s390x
+elixir:1.8-otp-22-alpine @ amd64
+elixir:1.8-otp-22-alpine @ arm32v7
+elixir:1.8-otp-22-alpine @ arm64v8
+elixir:1.8-otp-22-alpine @ i386
+elixir:1.8-otp-22-alpine @ ppc64le
+elixir:1.8-otp-22-alpine @ s390x
 elixir:1.8-slim @ amd64
 elixir:1.8-slim @ arm32v7
 elixir:1.8-slim @ arm64v8
@@ -104,18 +116,6 @@ elixir:latest @ arm64v8
 elixir:latest @ i386
 elixir:latest @ ppc64le
 elixir:latest @ s390x
-elixir:otp-22 @ amd64
-elixir:otp-22 @ arm32v7
-elixir:otp-22 @ arm64v8
-elixir:otp-22 @ i386
-elixir:otp-22 @ ppc64le
-elixir:otp-22 @ s390x
-elixir:otp-22-alpine @ amd64
-elixir:otp-22-alpine @ arm32v7
-elixir:otp-22-alpine @ arm64v8
-elixir:otp-22-alpine @ i386
-elixir:otp-22-alpine @ ppc64le
-elixir:otp-22-alpine @ s390x
 elixir:slim @ amd64
 elixir:slim @ arm32v7
 elixir:slim @ arm64v8
diff --git a/_bashbrew-list b/_bashbrew-list
index b1aae36..25890b0 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -37,11 +37,9 @@ elixir:1.8.2-slim
 elixir:1.9
 elixir:1.9-alpine
 elixir:1.9-slim
-elixir:1.9.0
-elixir:1.9.0-alpine
-elixir:1.9.0-slim
+elixir:1.9.1
+elixir:1.9.1-alpine
+elixir:1.9.1-slim
 elixir:alpine
 elixir:latest
-elixir:otp-22
-elixir:otp-22-alpine
 elixir:slim
diff --git a/elixir_otp-22-alpine/Dockerfile b/elixir_1.8-otp-22-alpine/Dockerfile
similarity index 100%
rename from elixir_otp-22-alpine/Dockerfile
rename to elixir_1.8-otp-22-alpine/Dockerfile
diff --git a/elixir_otp-22/Dockerfile b/elixir_1.8-otp-22/Dockerfile
similarity index 100%
rename from elixir_otp-22/Dockerfile
rename to elixir_1.8-otp-22/Dockerfile
diff --git a/elixir_alpine/Dockerfile b/elixir_alpine/Dockerfile
index 683851c..d7c0239 100644
--- a/elixir_alpine/Dockerfile
+++ b/elixir_alpine/Dockerfile
@@ -1,12 +1,12 @@
 FROM erlang:22-alpine
 
 # elixir expects utf8.
-ENV ELIXIR_VERSION="v1.9.0" \
+ENV ELIXIR_VERSION="v1.9.1" \
 	LANG=C.UTF-8
 
 RUN set -xe \
 	&& ELIXIR_DOWNLOAD_URL="https://github.com/elixir-lang/elixir/archive/${ELIXIR_VERSION}.tar.gz" \
-	&& ELIXIR_DOWNLOAD_SHA256="dbf4cb66634e22d60fe4aa162946c992257f700c7db123212e7e29d1c0b0c487" \
+	&& ELIXIR_DOWNLOAD_SHA256="94daa716abbd4493405fb2032514195077ac7bc73dc2999922f13c7d8ea58777" \
 	&& buildDeps=' \
 		ca-certificates \
 		curl \
diff --git a/elixir_latest/Dockerfile b/elixir_latest/Dockerfile
index 27df606..4401bba 100644
--- a/elixir_latest/Dockerfile
+++ b/elixir_latest/Dockerfile
@@ -1,12 +1,12 @@
 FROM erlang:22
 
 # elixir expects utf8.
-ENV ELIXIR_VERSION="v1.9.0" \
+ENV ELIXIR_VERSION="v1.9.1" \
 	LANG=C.UTF-8
 
 RUN set -xe \
 	&& ELIXIR_DOWNLOAD_URL="https://github.com/elixir-lang/elixir/archive/${ELIXIR_VERSION}.tar.gz" \
-	&& ELIXIR_DOWNLOAD_SHA256="dbf4cb66634e22d60fe4aa162946c992257f700c7db123212e7e29d1c0b0c487" \
+	&& ELIXIR_DOWNLOAD_SHA256="94daa716abbd4493405fb2032514195077ac7bc73dc2999922f13c7d8ea58777" \
 	&& curl -fSL -o elixir-src.tar.gz $ELIXIR_DOWNLOAD_URL \
 	&& echo "$ELIXIR_DOWNLOAD_SHA256  elixir-src.tar.gz" | sha256sum -c - \
 	&& mkdir -p /usr/local/src/elixir \
diff --git a/elixir_slim/Dockerfile b/elixir_slim/Dockerfile
index 202ec4c..cf4fb4a 100644
--- a/elixir_slim/Dockerfile
+++ b/elixir_slim/Dockerfile
@@ -1,12 +1,12 @@
 FROM erlang:22-slim
 
 # elixir expects utf8.
-ENV ELIXIR_VERSION="v1.9.0" \
+ENV ELIXIR_VERSION="v1.9.1" \
 	LANG=C.UTF-8
 
 RUN set -xe \
 	&& ELIXIR_DOWNLOAD_URL="https://github.com/elixir-lang/elixir/archive/${ELIXIR_VERSION}.tar.gz" \
-	&& ELIXIR_DOWNLOAD_SHA256="dbf4cb66634e22d60fe4aa162946c992257f700c7db123212e7e29d1c0b0c487" \
+	&& ELIXIR_DOWNLOAD_SHA256="94daa716abbd4493405fb2032514195077ac7bc73dc2999922f13c7d8ea58777" \
 	&& buildDeps=' \
 		ca-certificates \
 		curl \
```

</details>

File: library/elixir [hunk 1]
```diff
@@ -33,12 +33,12 @@ Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
...
 Directory: 1.8/alpine
 
-Tags: 1.8.2-otp-22, 1.8-otp-22, otp-22
+Tags: 1.8.2-otp-22, 1.8-otp-22
 Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
 GitCommit: 6dc5ffd3b4c2915096887b45ba8e71d391ce2398
 Directory: 1.8/otp-22
 
-Tags: 1.8.2-otp-22-alpine, 1.8-otp-22-alpine, otp-22-alpine
+Tags: 1.8.2-otp-22-alpine, 1.8-otp-22-alpine
```
Comment (yosifkit) [inline library/elixir]: Are the `elixir:otp-22` and `elixir:otp-22-alpine` tags meant to be removed?

(just double checking that this is the desired outcome)

Comment (getong) [reply library/elixir]: I recheck the code, the 1.9 branch is the current latest branch, and it complie with otp 22. No need the otp-22-alpine, just the alpine is fine.
But the 1.8 is not the latest branch.
So I think the code is correct.

**Comment by yosifkit** [comment]
Build test of #6370; 9b38897c; `amd64` (`elixir`):

```console

$ bashbrew build elixir:1.9.1
Using bashbrew/cache:4014dcb1cfea40ce60db6480654539f5610c99266ce7b09739cd58665bfbdc62 (elixir:1.9.1)
Tagging elixir:1.9.1
Tagging elixir:1.9
Tagging elixir:latest

$ test/run.sh elixir:1.9.1
testing elixir:1.9.1
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.9.1-slim
Using bashbrew/cache:a09fd8797ce7901ae84f3b83a9512f09bcef01ba0bdae0bb6de8983b139f73b4 (elixir:1.9.1-slim)
Tagging elixir:1.9.1-slim
Tagging elixir:1.9-slim
Tagging elixir:slim

$ test/run.sh elixir:1.9.1-slim
testing elixir:1.9.1-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.9.1-alpine
Using bashbrew/cache:0f57cdae5472afaca9ecc62933069469e91dc249af2362674cf79555b32a70b7 (elixir:1.9.1-alpine)
Tagging elixir:1.9.1-alpine
Tagging elixir:1.9-alpine
Tagging elixir:alpine

$ test/run.sh elixir:1.9.1-alpine
testing elixir:1.9.1-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.8.2
Using bashbrew/cache:488e08c47243cfa9622415734f64281c58c8de122b6258c9e5ea4a1f37d27c58 (elixir:1.8.2)
Tagging elixir:1.8.2
Tagging elixir:1.8

$ test/run.sh elixir:1.8.2
testing elixir:1.8.2
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.8.2-slim
Using bashbrew/cache:15e4c6c7ba7fec7008ae6f4546c0e11d87b80847daea943d5402c634b1f07a65 (elixir:1.8.2-slim)
Tagging elixir:1.8.2-slim
Tagging elixir:1.8-slim

$ test/run.sh elixir:1.8.2-slim
testing elixir:1.8.2-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.8.2-alpine
Using bashbrew/cache:b497a0cbbf457522285bc389e1016fb81b71b3de1931eb041604b3a2701ddd29 (elixir:1.8.2-alpine)
Tagging elixir:1.8.2-alpine
Tagging elixir:1.8-alpine

$ test/run.sh elixir:1.8.2-alpine
testing elixir:1.8.2-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.8.2-otp-22
Using bashbrew/cache:345654403aa2a98fcc3c74c0fe3d0baa7bb773418b5b8f7b85064ffcf484ee37 (elixir:1.8.2-otp-22)
Tagging elixir:1.8.2-otp-22
Tagging elixir:1.8-otp-22

$ test/run.sh elixir:1.8.2-otp-22
testing elixir:1.8.2-otp-22
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.8.2-otp-22-alpine
Using bashbrew/cache:96301c526a029d69bab87fed1c075a1802b218292ec490f748e21a9003bef8a1 (elixir:1.8.2-otp-22-alpine)
Tagging elixir:1.8.2-otp-22-alpine
Tagging elixir:1.8-otp-22-alpine

$ test/run.sh elixir:1.8.2-otp-22-alpine
testing elixir:1.8.2-otp-22-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.7.4
Using bashbrew/cache:30f401029063c159d9cac4c9c7a68714b3977c6661d2de82a3b1885c0879ca0c (elixir:1.7.4)
Tagging elixir:1.7.4
Tagging elixir:1.7

$ test/run.sh elixir:1.7.4
testing elixir:1.7.4
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.7.4-slim
Using bashbrew/cache:2dd851fd07a50d7dda9082ad15d659a5b81a0bc324a929934d2832b243f10c19 (elixir:1.7.4-slim)
Tagging elixir:1.7.4-slim
Tagging elixir:1.7-slim

$ test/run.sh elixir:1.7.4-slim
testing elixir:1.7.4-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.7.4-alpine
Using bashbrew/cache:9adb686d26106eeabff6cba1ffabeb8004d8b8d40125aee4d5f237fda80c47e5 (elixir:1.7.4-alpine)
Tagging elixir:1.7.4-alpine
Tagging elixir:1.7-alpine

$ test/run.sh elixir:1.7.4-alpine
testing elixir:1.7.4-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.6.6
Using bashbrew/cache:fd1991122a287dbadc2e43704f229d47246e129b83e0229d78c278cd39b3c569 (elixir:1.6.6)
Tagging elixir:1.6.6
Tagging elixir:1.6

$ test/run.sh elixir:1.6.6
testing elixir:1.6.6
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.6.6-slim
Using bashbrew/cache:e083c8174e98a7f030a60a6d3d4efc8edd525bb2ac8717696ad8c5754e3686ed (elixir:1.6.6-slim)
Tagging elixir:1.6.6-slim
Tagging elixir:1.6-slim

$ test/run.sh elixir:1.6.6-slim
testing elixir:1.6.6-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.6.6-alpine
Using bashbrew/cache:7d6bdd866f50738d8e2cc9fbf2c849ccef966c6a6414dec237e16e3068b53c46 (elixir:1.6.6-alpine)
Tagging elixir:1.6.6-alpine
Tagging elixir:1.6-alpine

$ test/run.sh elixir:1.6.6-alpine
testing elixir:1.6.6-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.6.6-otp-21
Using bashbrew/cache:d289aade70e04612f788314d3aba6293cc58b4a5eaa91a6df019e7fc5759ba21 (elixir:1.6.6-otp-21)
Tagging elixir:1.6.6-otp-21
Tagging elixir:1.6-otp-21

$ test/run.sh elixir:1.6.6-otp-21
testing elixir:1.6.6-otp-21
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.6.6-otp-21-alpine
Using bashbrew/cache:dff3b93f0007fa7f8d2bf3c75d38220079cf49aa3c0f8df102f56dcc71f13458 (elixir:1.6.6-otp-21-alpine)
Tagging elixir:1.6.6-otp-21-alpine
Tagging elixir:1.6-otp-21-alpine

$ test/run.sh elixir:1.6.6-otp-21-alpine
testing elixir:1.6.6-otp-21-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.5.3
Using bashbrew/cache:040e85b73dba2d1161e9f7ded2363c2ab0f26e60359374fbdc7f36e7cd248ff3 (elixir:1.5.3)
Tagging elixir:1.5.3
Tagging elixir:1.5

$ test/run.sh elixir:1.5.3
testing elixir:1.5.3
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.5.3-slim
Using bashbrew/cache:ba0e032b70748c20efcb908b5e631cccee2c0aeee44f8416b3f790df6e14b75f (elixir:1.5.3-slim)
Tagging elixir:1.5.3-slim
Tagging elixir:1.5-slim

$ test/run.sh elixir:1.5.3-slim
testing elixir:1.5.3-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.5.3-alpine
Using bashbrew/cache:4b1b35411a34ae1a4ec1486c777fde12411b135847e858591f45427b3d60504d (elixir:1.5.3-alpine)
Tagging elixir:1.5.3-alpine
Tagging elixir:1.5-alpine

$ test/run.sh elixir:1.5.3-alpine
testing elixir:1.5.3-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.4.5
Using bashbrew/cache:97a9ba7e3ff462858dc1ed32c6935889450b934ec7862a7ffe18d43ee99c8b44 (elixir:1.4.5)
Tagging elixir:1.4.5
Tagging elixir:1.4

$ test/run.sh elixir:1.4.5
testing elixir:1.4.5
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed


$ bashbrew build elixir:1.4.5-slim
Using bashbrew/cache:cda4756a98f678027cee1d50b7eac7ef5249f38663713ce4ba792458c01ff30b (elixir:1.4.5-slim)
Tagging elixir:1.4.5-slim
Tagging elixir:1.4-slim

$ test/run.sh elixir:1.4.5-slim
testing elixir:1.4.5-slim
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'elixir-hello-world' [5/5]...passed

```

**Pull request merged**